CRM/Contact/Form/Relationship.php

```diff
@@ -445,6 +450,14 @@ public function postProcess() {
         // clear the current employer. CRM-3235.
         $relChanged = $params['relationship_type_id'] != $this->_values['relationship_type_id'];
         if (!$params['is_active'] || !$params['is_current_employer'] || $relChanged) {
+
+          // CRM-15881 UPDATES
+          // If not is_active then is_current_employer needs to be set false as well! Logically a contact cannot be a current employee of a disabled employer relationship.
+          // If this is not done, then the below process will go ahead and disable the organization_name and employer_id fields in the contact record (which is what is wanted) but then further down will be re-enabled becuase is_current_employer is not false, therefore undoing what was done correctly.
+          if(!$params['is_active']) {
```

Comment:
@marcov4lente you need to leave a space after "if" statement. Can you please fix the same